### PR TITLE
Clean up some link styling css

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

This PR starts cleaning up some CSS rules, so it will be easier to work on #20706
I'll try to create multiple smaller PRs to clean up first, so it's easier to review each.
Afterwards I hope adjusting the links will be a bit simpler...

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by michalkleiner — APPROVED** [review]

**Comment by michalkleiner** [comment]
Probably approved too early. Let me know if you plan on adding more and I can look at it again later.

**Comment by sgiehl** [comment]
There had been some failures in the UI tests. Trying to clean up a bit in order to fix that properly

**Comment by sgiehl** [comment]
@michalkleiner this should be ready for review now. I've adjusted a couple more things to clean up the css a bit.
I've also unified the definitions for links globally, so they by default look the same all across Matomo. This also means that for now in the Installation and in the Updater the links are no longer underlined. That will be readded when we change the global links design later again.

File: plugins/CoreHome/stylesheets/dataTable/_dataTable.less [hunk 1]
```diff
@@ -329,20 +329,22 @@ table.dataTable img {
 }
 
 .datatableRelatedReports {
-  color: #888;
+  color: #808080;
```
Comment (michalkleiner) [inline plugins/CoreHome/stylesheets/dataTable/_dataTable.less:332]: Minor tweak but technically 888 becomes 888888 unless it's to also address contrast issue etc.
```suggestion
  color: #888888;
```

File: plugins/CoreHome/stylesheets/coreHome.less [hunk 2]
```diff
@@ -163,7 +163,7 @@ div.pk-emptyGraph {
 
 .Piwik_Popover_Loading_Subject {
     padding: 0 70px 55px 70px;
-    color: #7e7363;
+    color: @theme-color-link!important;
```
Comment (michalkleiner) [inline plugins/CoreHome/stylesheets/coreHome.less]:
```suggestion
    color: @theme-color-link !important;
```

File: plugins/Morpheus/stylesheets/general/_default.less [hunk 3]
```diff
@@ -119,17 +119,18 @@ body {
...
-}
-
 #root {
   margin: 0 10px;
   position: relative;
 }
 
+a {
+ color: @theme-color-link;
+ text-decoration: none;
```
Comment (michalkleiner) [inline plugins/Morpheus/stylesheets/general/_default.less:129]: I'm torn here whether we should have kept it as underlined as it was, does it help anything removing that now and re-add it later?

File: plugins/CoreHome/stylesheets/dataTable/_dataTable.less (hunk 1 above)
Comment (sgiehl) [reply plugins/CoreHome/stylesheets/dataTable/_dataTable.less:332]: That one had been overwritten somewhere else. I just combined the two rules.

File: plugins/Morpheus/stylesheets/general/_default.less (hunk 3 above)
Comment (sgiehl) [reply plugins/Morpheus/stylesheets/general/_default.less:129]: There had been one global rule defining it as underlined. And another rule in a css file that isn't included in the installation css, defining it as not underlined. This PR unifies both to one global rule. Reducing that now will make it easier to change the link styles later globally.

**Pull request merged**